src/rubocop.ts

```diff
@@ -87,13 +87,22 @@ function getCurrentPath(fileUri: vscode.Uri): string {
 function getCommandArguments(fileName: string): string[] {
   let commandArguments = ['--stdin', fileName, '--force-exclusion'];
   const extensionConfig = getConfig();
+
   if (extensionConfig.configFilePath !== '') {
-    const found = [extensionConfig.configFilePath]
+    const expandedConfigFilePath = path.isAbsolute(
+      extensionConfig.configFilePath
+    )
+      ? extensionConfig.configFilePath
+      : path.resolve(extensionConfig.configFilePath);
```

Comment:
@Skipants like this right?

```suggestion
    const expandedConfigFilePath = path.resolve(extensionConfig.configFilePath);
```

Reply:
@TSMMark Yup! I really should have just used the suggestion feature myself, huh? 🤦

Reply:
It's a great feature! 

I'll commit it